feature-libs/product-configurator/rulebased/components/attribute/product-card/configurator-attribute-product-card.component.ts

```diff
@@ -0,0 +1,175 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  Input,
+  OnInit,
+  Output,
+} from '@angular/core';
+import { Configurator } from '../../../core/model/configurator.model';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { QuantityUpdateEvent } from '../../form/configurator-form.event';
+import { Product, ProductService } from '@spartacus/core';
+import { map } from 'rxjs/operators';
+import {
+  ConfiguratorAttributeQuantityComponentOptions,
+  Quantity,
+} from '../quantity/configurator-attribute-quantity.component';
+import { ConfiguratorPriceComponentOptions } from '../../price/configurator-price.component';
+
+interface ProductExtended extends Product {
+  noLink?: boolean;
+}
+
+export interface ConfiguratorAttributeProductCardComponentOptions {
+  preventAction?: boolean;
+  multiSelect?: boolean;
+  product?: Configurator.Value;
```

Comment:
As this doesn't need to always represent a product, the name should be different